stdlib/strings/strings_test.flux

```diff
@@ -0,0 +1,218 @@
+"
+    result = testing.loadStorage(csv: inData)
+		|> range(start: 2018-05-22T19:53:26Z)
+		|> map(fn: (r) =>
+        			({r with _value: strings.replace(v: r._value, t: "LnSk", u: " ", i: 1)}))
+    testing.diff(got: result, want: testing.loadMem(csv: outData))
+}
+
+testcase string_substring {
+    outData = "
+#datatype,string,long,dateTime:RFC3339,dateTime:RFC3339,dateTime:RFC3339,string,string,string,string,string,string,string,string
+#group,false,false,true,true,false,false,true,true,true,true,true,true,false
+#default,_result,,,,,,,,,,,,
+,result,table,_start,_stop,_time,_value,_field,_measurement,device,fstype,host,path,sub
+,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-22T19:53:26Z,字,used_percent,disk,disk1,apfs,host.local,/,字
+,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-22T19:53:36Z,k9n  gm,used_percent,disk,disk1,apfs,host.local,/,m
+,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-22T19:53:46Z,b  ,used_percent,disk,disk1,apfs,host.local,/, 
+,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-22T19:53:56Z,2COTDe,used_percent,disk,disk1,apfs,host.local,/,e
+,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-22T19:54:06Z,cLnSkNMI  ,used_percent,disk,disk1,apfs,host.local,/, 
+,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-22T19:54:16Z,13F2  ,used_percent,disk,disk1,apfs,host.local,/, 
+"
+    result = testing.loadStorage(csv: inData)
+		|> range(start: 2018-05-22T19:53:26Z)
+		|> map(fn: (r) =>
+        			({r with sub: strings.substring(v: r._value, start: strings.strlen(v: r._value)-1, end: strings.strlen(v: r._value))}))
+    testing.diff(got: result, want: testing.loadMem(csv: outData))
+}
+
+testcase string_length {
+    outData = "
+#datatype,string,long,dateTime:RFC3339,dateTime:RFC3339,dateTime:RFC3339,string,string,string,string,string,string,string,long
+#group,false,false,true,true,false,false,true,true,true,true,true,true,false
+#default,_result,,,,,,,,,,,,
+,result,table,_start,_stop,_time,_value,_field,_measurement,device,fstype,host,path,len
+,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-22T19:53:26Z,字,used_percent,disk,disk1,apfs,host.local,/,1
+,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-22T19:53:36Z,k9n  gm,used_percent,disk,disk1,apfs,host.local,/,7
+,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-22T19:53:46Z,b  ,used_percent,disk,disk1,apfs,host.local,/,3
+,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-22T19:53:56Z,2COTDe,used_percent,disk,disk1,apfs,host.local,/,6
+,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-22T19:54:06Z,cLnSkNMI  ,used_percent,disk,disk1,apfs,host.local,/,10
+,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-22T19:54:16Z,13F2  ,used_percent,disk,disk1,apfs,host.local,/,6
+"
+    result = testing.loadStorage(csv: inData)
+		|> range(start: 2018-05-22T19:53:26Z)
+		|> map(fn: (r) =>
+        			({r with len: strings.strlen(v: r._value)}))
+    testing.diff(got: result, want: testing.loadMem(csv: outData))
+}
```

Comment:
Just watch out for these lack of newline messages. Vim and other editors will ensure that all files always end with a newline, but some editors don't. There's usually a setting to enable this. It's just nice for standardization if files are committed with the standard newline file ending.